kuma/users/views.py

```diff
@@ -796,7 +828,11 @@ def create_stripe_subscription(request):
     try:
         email = request.POST.get("stripe_email", "")
         stripe_token = request.POST.get("stripe_token", "")
-        create_stripe_customer_and_subscription_for_user(user, email, stripe_token)
+        subscription = create_stripe_customer_and_subscription_for_user(
+            user, email, stripe_token
+        )
+        UserSubscription.set_active(user, subscription.id)
```

Comment:
What I'd *like* to do is to not do it like this but instead depend on signals. But I'm not sure it's a great win either way.